.github/workflows/main.yml

```diff
@@ -9,9 +9,8 @@ on:
 
 env:
   REGISTRY: ghcr.io
-  IMAGE_NAME: ghcr.io/${{ github.repository }}
+  IMAGE_NAME: ghcr.io/genomicdatainfrastructure/gdi-userportal-frontend
   AZURE_WEBAPP_NAME: catalogue-test
-  DOCKER_METADATA_OUTPUT_TAGS:
```

Comment:
I think there are more references to `DOCKER_METADATA_OUTPUT_TAGS`, can you please replace all of them with IMAGE_NAME?